recipes/sqlite3/all/test_package/conanfile.py

```diff
@@ -7,11 +7,19 @@ class TestPackageConan(ConanFile):
     generators = "cmake", "cmake_find_package"
 
     def build(self):
+        if not self.options["sqlite3"].enable_default_vfs:
+            # Need to provide custom VFS code: https://www.sqlite.org/custombuild.html
```

Comment:
Would it be possible to create an empty/dummy vfs?
See https://www.sqlite.org/c3ref/vfs.html
Or is this out of scope?

At the end, the only thing we care about is whether the library can be linked against and whether basic functionality is there.

Reply:
Nowadays that implementation is out of the scope.

You can link created library if you provide `sqlite3_os_init` and `sqlite3_os_end` function implementations ([that document](https://www.sqlite.org/custombuild.html), chapter 5.0)

That's useful for cross-compiled embedded devices, where you need to use custom FS API access instead of libc `fopen`, `fclose` and others.

Reply:
Great dummy vfs implementation you've written!
The only thing I'm worried about now is how it will behave when the layout of the vfs struct changes. I guess the sqlite developers guarantee some stability?

Reply:
SQLite developers use versions for that. I implemented version 2 instead of the latest 3.

Reply:
Superbe!
Then I'll wait on your EAP approval in #4 and for ci to finish to approve this change.